### PR TITLE
Add links to every HTTP services found into the Symfony WDT

### LINKED ISSUE #197: Add Kibana to toolbar if detected by Docker integration?
Hi,

I'm wondering if it is possible to add Kibana in the toolbar if it's detected by Docker integration?

It would be nice as Elasticsearch is already handled by the CLI :)

### PR DESCRIPTION
**Pull request by tucksaun**
Fix #197 by providing an even more generic solution.
based on work made in #263, as such Draft until #263 is merged

Currently, this looks like this:
Screenshot 2023-02-01 at 17 00 43
@fabpot @jmsche @javiereguiluz any help on the look is welcome :)

### CODE REVIEW

**Comment by fabpot** [comment]
For the design, maybe we can make the title a link and remove the "Open" link (for the new section, but also for the existing one).

**Comment by jmsche** [comment]
Nice idea from @fabpot :) 

If applied, the link should be added into the `<b>` tag, as putting it around this tag would not render nicely.

**Comment by tucksaun** [comment]
I like the idea @fabpot 
What about the status badge? should we keep them or remove all of them?

**Comment by tucksaun** [comment]
New look:
Screenshot 2023-02-02 at 16 24 03

File: local/php/toolbar.go [hunk 1]
```diff
@@ -81,23 +82,32 @@ func (p *Server) tweakToolbar(body io.ReadCloser, env map[string]string) (io.Rea
...
 	blackfire := `<span class="sf-toolbar-status sf-toolbar-status-red">Down</span>`
+	extraLinks := ``
 	if env, err := envs.NewLocal(p.projectDir, terminal.IsDebug()); err == nil {
-		values := envs.AsMap(env)
-		if prefix := env.FindRelationshipPrefix("mailer", "http"); prefix != "" {
-			if url, exists := values[prefix+"URL"]; exists {
-				webmail = fmt.Sprintf(`<span class="sf-toolbar-status sf-toolbar-status-green">Up</span>&nbsp;&nbsp;&nbsp;<a class="sf-cli-webmail" href="%s" target="_blank">Open</a>`, url)
-			}
+		if url, exists := env.FindServiceUrl("mailer"); exists {
+			webmail = fmt.Sprintf(`<b><a class="sf-cli-webmail" href="%s" target="_blank">Webmail</a></b> <span class="sf-toolbar-status sf-toolbar-status-green">Up</span></a>`, url)
```
Comment (javiereguiluz) [inline local/php/toolbar.go:91]: For security reasons, we could add `rel="noopener"` to all `target="_blank"` links. Some browsers like Chrome do that automatically, but others don't do that yet.

See https://developer.chrome.com/docs/lighthouse/best-practices/external-anchors-use-rel-noopener/

Comment (fabpot) [reply local/php/toolbar.go:91]: Works for me.

Comment (tucksaun) [reply local/php/toolbar.go]: makes sense 👍 
done

**Pull request merged**